File: R5J4/test/j20439_No0421_1.c
#include <stdio.h>

int main() {
  int n;
  int ima=0;
  printf("Input: ");
  scanf("%d", &n);
  for (int i=1; i<=n; i++) {
	 if (i%3 == 0) {
		printf("%d いまでしょ！\n", i);
		ima=1;
	 }
	 int m = i;
	 while (m!=0) {
		if (m%10 == 3) {
		  printf("%d いまでしょ！\n", i);
		  ima=1;
		  break;
		}
		m /= 10;
	 }
	 if (ima == 0) {
		printf("%d いつやるか？\n", i);
	 }
	 ima=0;
  }
}

File: R5J4/test/j20439_No0421_1.py
n = int(input("Input: "))
for i in range(1, n+1):
    if i % 3 == 0:
        print("%d いまでしょ！" %(i))
        continue
    if '3' in str(i):
        print("%d いまでしょ！" %(i))
        continue
    print("%d いつやるか？" %(i))
